src/Microsoft.ApiDesignGuidelines.Analyzers/Core/PassSystemUriObjectsInsteadOfStrings.cs

```diff
@@ -24,15 +29,202 @@ public abstract class PassSystemUriObjectsInsteadOfStringsAnalyzer : DiagnosticA
                                                                              s_localizableMessage,
```

Comment:
You need to change the context of `PassSystemUriObjectsInsteadOfStringsMessage` to reflect what FXCop was emitting. That is instead of the current string `Pass system uri objects instead of strings`, it should be the text [here](http://index/#UsageRules/PassSystemUriObjectsInsteadOfStrings.cs,73) with 3 format arguments.